src/main.cpp

```diff
@@ -819,4 +827,21 @@ bool runUpdate()
     }
     return false;
 }
-#endif
+#endif // defined(Q_OS_WINDOWS) && defined(INCLUDE_UPDATER)
+
+void initializeQRCResources()
+{
+    #ifdef INCLUDE_VARIABLE_SPLASH_SCREEN
```

Comment:
Our coding style does NOT indent preprocessor directives - and particularly where they are nested (like starting on line 840) there is merit in commenting `#endif`s to clarify the scope (instead of using indentation).